user-manual/connectors/source/mysql.md

```diff
@@ -1,7 +1,132 @@
 ---
 title: MySQL
-id: mysql
 ---
 
-# MySQL
-Coming soon
\ No newline at end of file
+# MySql Source
+This document provides a brief introduction of the MySQL Source.
+It is also designed to guide you through the process of running an
+MySQL Source Connector.
+## Introduction
+The MySql Source is a [Vance Connector][vc] that use [Debezium][debezium] 
+to obtain a snapshot of the existing data in a MySql database and then 
+monitors and record all subsequent row-level changes to that data.
+
+For example, MySql database vance has table vance_test Look:
+
+```text
++-------------+--------------+------+-----+---------+----------------+
+| Field       | Type         | Null | Key | Default | Extra          |
++-------------+--------------+------+-----+---------+----------------+
+| id          | int          | NO   | PRI | NULL    | auto_increment |
+| name        | varchar(100) | NO   |     | NULL    |                |
+| description | varchar(100) | NO   |     | NULL    |                |
+| date        | date         | YES  |     | NULL    |                |
++-------------+--------------+------+-----+---------+----------------+
+```
+
```

Comment:
I think it needs to tell the user, the connector will transform MySQL record to a CloudEvent like below